prisma/schema.prisma

```diff
@@ -2,6 +2,11 @@ generator client {
   provider = "prisma-client-js"
 }
 
+// generator typegraphql {
```

Comment:
@Jonasdoubleyou I guess this is commented out since the generated GraphQL code is committed? In order to update that code (when the data model changes) does one simply has to uncomment this in order to generate the new GraphQL endpoints?

Reply:
exactly!